snuba/utils/streams/kafka.py

```diff
@@ -58,12 +58,30 @@ def __init__(self, state: KafkaConsumerState):
         self.__state = state
 
 
-@dataclass(frozen=True)
+Headers = Sequence[Tuple[str, bytes]]
```

Comment:
This maps to the return value of `Message.headers()` as documented here: https://docs.confluent.io/current/clients/confluent-kafka-python/#confluent_kafka.Message.headers

More detail here on the general design here: https://cwiki.apache.org/confluence/display/KAFKA/KIP-82+-+Add+Record+Headers